### ISSUE TITLE
Dont know how to make it work for CoC

### ISSUE THREAD

**Issue by lozalojo** [theripper93/Splatter#7]
System: https://github.com/HavlockV/CoC7-FoundryVTT

Using lastest system and Core Foundry.

Data I've entered is:

data.type
data.type
data.attribs.hp.value
data.attribs.hp.max

And doesnt seem to work, any clues?

**Comment by theripper93** [comment]
you will have better luck asking in the coc discord channel, i don't know all the data path for all systems :P

**Comment by lozalojo** [comment]
I am aware of the CoC actor structure, what I didnt know was if the settings were relative path (to data or to data.data) or abs path.

![](https://i.ibb.co/37wnnWp/screenshot-20.jpg)

Finally I made it work, looks like I needed to refresh after updating the settings:

![](https://i.ibb.co/Y06scSV/screenshot-21.jpg)
![](https://i.ibb.co/B3W58Dj/screenshot-113.jpg)

What I have not succeed is with the macro config, I've made a macro:

```js
const BloodSheetData = {
  "creature": "#325835da",
  "character": "#b527d5d5"
};
game.settings.set("splatter", "BloodSheetData", BloodSheetData);```

I dont know if I have to run once per combat, once in each world, once and period, everytime I want to modify colours...

All in all, after executing the macro some times, I cant get to distinguish between "character" and "creature" in terms of blood color

**Comment by theripper93** [comment]
the macro just sets a hidden game setting, so just run it once per world.
That said, i cannot see very well but i think your type is in data.type - the settings are relative to actor.data

**Comment by lozalojo** [comment]
> the macro just sets a hidden game setting, so just run it once per world.

Oh, ok, thanks, so once per world, perfect.

> That said, i cannot see very well but i think your type is set correctly - the settings are relative to actor.data.

Nope, abs paths are:

actor.data.type
actor.data.data.attribs.hp.value
actor.data.data.attribs.hp.max

So _type_ might work, In fact, I've tested the macro giving different colours to characters, creatures and npcs, and _splatter_ detects the actor types and shows blood accordingly the color just configured.

Thanks for your work! I am eager to see in play.

Also, I've just translated to spanish, I have to test it this afternoon and I will PR afterwards.

> Try to give it something non existent to the custom type like just put "wsertf4wr23qf23wf32w" in the setting field

What's for? I suppose it is for some other system that uses a field to customize creatures names in the sheet.
